base/linalg/blas.jl

```diff
@@ -1044,31 +1105,27 @@ end
 """
     symm(side, ul, alpha, A, B)
 
-Returns `alpha*A*B` or `alpha*B*A` according to `side`. `A` is assumed to be symmetric. Only
-the `ul` triangle of `A` is used.
+Returns `alpha*A*B` or `alpha*B*A` according to [`side`](@ref stdlib-blas-side).
+`A` is assumed to be symmetric. Only
+the [`ul`](@ref stdlib-blas-uplo) triangle of `A` is used.
 """
 symm(side, ul, alpha, A, B)
 
 """
     symm(side, ul, A, B)
 
-Returns `A*B` or `B*A` according to `side`. `A` is assumed to be symmetric. Only the `ul`
+Returns `A*B` or `B*A` according to [`side`](@ref stdlib-blas-side).
+`A` is assumed to be symmetric. Only the [`ul`](@ref stdlib-blas-uplo)
 triangle of `A` is used.
 """
 symm(side, ul, A, B)
 
 """
-    symm(tA, tB, alpha, A, B)
```

Comment:
It doesn't really make sense for the _symmetric_ matrix multiply to take two transpose args. Also, it's not present in the Netlib docs.